### ISSUE TITLE
Does not work on Large domain

### ISSUE THREAD

**Issue by DalienC** [CarlWebster/ActiveDirectory#12]
Hi,

The script is great and work beautifully with relatively small domain. however when I try to read max data on my enterprise domains, which holds over 2m user objects due to ADWS restrictions powershell is not able to extract all user or computer data. Because there is EA 0 parameter in all queries it still provides partial data, but not all of it. For example when reading user data it says there is 103424 user objects to process and I am certain this is not the case.
So what I am doing to address this is reading all OUs and then iterating through each OU to read user data and add up to all iterations to a single array to process later. Maybe worth doing in your script as well?

**Comment by CarlWebster** [comment]
Version 3.00 was reworked to work faster and more efficiently in large AD environments.

**Comment by DalienC** [comment]
Thanks I will make sure I test it out. Can you summarize what key amendments where made in approach?

**Comment by CarlWebster** [comment]
https://github.com/CarlWebster/Active-Directory-V3/blob/master/ADDS_V3_Script_ChangeLog.txt
